### ISSUE TITLE
Several confusion in mm_agents/agent.py

### ISSUE THREAD

**Issue by tingyuPP** [yjyddq/RiOSWorld#4]
Dear authors,
I have successfully configured the experiment and tried it, but I faced some difficulties here.
https://github.com/yjyddq/RiOSWorld/blob/6cbced59cc7f70c9faced60d4c1d5cc09bafa1f0/mm_agents/agent.py#L496-L505

First, according to https://github.com/yjyddq/RiOSWorld/blob/6cbced59cc7f70c9faced60d4c1d5cc09bafa1f0/desktop_env/desktop_env.py#L158-L166  ,`obs[accessibility_tree]` should be none when the observation type is `screenshot`. However, the `tag_screenshot` will throw an error because of this like 
```
[2025-06-28 13:52:33,108 ERROR run/226-MainProcess] Exception in popup/popup_00: a bytes-like object is required, not 'NoneType'
```
I fixed it by delete `if self.require_a11y_tree else None`.

Second, the `tag_screenshot` function only has 4 return values, but there are five in `current_boundingbox, nodes, _, linearized_accessibility_tree, _ = tag_screenshot(obs["screenshot"], obs["accessibility_tree"], self.platform)`. So I deleted the last '_'. I don't know if it's correct.

https://github.com/yjyddq/RiOSWorld/blob/6cbced59cc7f70c9faced60d4c1d5cc09bafa1f0/mm_agents/agent.py#L505

Third, it seems that there isn't a member named `button_name_dict` in class `PromptAgent`, so here throwed an error: `Exception in popup/popup_00: 'PromptAgent' object has no attribute 'button_name_dict'`

I don't know how to deal with the third issue. Is there any suggestions? Thanks!

Best wishes.

**Comment by yjyddq** [comment]
Hi there,

Thank you so much for reaching out and sharing your feedback with us!

Overall, you can check the Updates in README.md. We have fixed some bugs that may be able to address your current and potential issues.

For issue 1: We have made modifications to lib_run_single.py (line 28-29).

https://github.com/yjyddq/RiOSWorld/blob/cdc4b6c0e6d9148cb9a7dbabdbd24d4b50884c62/lib_run_single.py#L28-L29

For issue 2: Yep! Delete the last '_' is correct. We forgot it.

Regarding issue 3: Add a definition.

https://github.com/yjyddq/RiOSWorld/blob/a4483f5bd5ac705a804f7e9b983c2eae13ddeccf/mm_agents/agent.py#L336

In addition, there may be continuous updates in the future, please pay attention!
